resources/build/tests/trees/.gitignore

```diff
@@ -0,0 +1 @@
+child?.*
```

Comment:
Why do we ignore these? They are checked in after all...

Reply:
Only the folders are checked in, but the scripts generate `child1.configure`, `child2.build`, etc etc. The folders also don't match because of the `.`.